microsoft-edge/devtools-guide-chromium/whats-new/2022/05/devtools-102.md

```diff
@@ -0,0 +1,201 @@
+---
+title: What's new in DevTools (Microsoft Edge 102)
+description: "DevTools extension: browser preview device/mode emulation and sourcemaps to debug/edit Sass/React.  Save heap snapshot strings to JSON.  Retainers filters in Memory tool.  Inspect tooltip persists.  Better high-contrast display.  Experiments have Help icons.  Improved Network tool layout. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 05/24/2022
+---
+# What's New in DevTools (Microsoft Edge 102)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
```

Comment:
In addition to the note, can we add a Tip for Build specifically for this What's New like we did here: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools?

```suggestion
> [!TIP]
> The **Microsoft Build 2022** conference was on May 24-26.  Learn more about new features for memory, performance, and production debugging in the DevTools and new capabilities in the web platform for PWAs in the following videos:
> [Tips for debugging memory, performance, & production issues](https://www.youtube.com/watch?v=hdrR0QwXpuc)
> [Microsoft Edge | Build 2022: Enriching desktop experiences with the power and reach of the web](https://www.youtube.com/watch?v=ahO5nePl4BQ)
```